tools/c7n_azure/tests_azure/tests_resources/test_storage.py

```diff
@@ -650,6 +650,22 @@ def _get_storage_management_client_api_string(self):
             .client('azure.mgmt.storage.StorageManagementClient')\
             .DEFAULT_API_VERSION.replace("-", "_")
 
+    def test_storage_soft_delete(self):
+        p = self.load_policy({
+            'name': 'test-azure-storage-blob',
+            'resource': 'azure.storage',
+            'filters': [
+                {'type': 'value',
+                'key': 'name',
+                'op': 'glob',
+                'value_type': 'normalize',
+                'value': 'cctstorage*'},
+                {'type': 'soft-delete',
```

Comment:
i think we're missing the filter implementation in the pr.

Reply:
I added the filter.